docs/docs/operate/autoscalingstrategies.md

```diff
@@ -0,0 +1,145 @@
+| Resource        | Metric  |
+| --------------- | ------- |
+| `CPU`           | group: `ResourceUsage` name: `cpuPctUsageCurr` aggregation: `AVG`    |
+| `Memory`        | group: `ResourceUsage` name: `totMemUsageCurr` aggregation: `AVG`    |
+| `Network`       | group: `ResourceUsage` name: `nwBytesUsageCurr` aggregation: `AVG`   |
+| `JVMMemory`     | group: `ResourceUsage` name: `jvmMemoryUsedBytes` aggregation: `AVG` |
+| `DataDrop`      | group: `DataDrop` name: `dropCount` aggregation: `AVG` |
+| `KafkaLag`      | group: `consumer-fetch-manager-metrics` name: `records-lag-max` aggregation: `MAX` |
+| `KafkaProcessed` | group: `consumer-fetch-manager-metrics` name: `records-consumed-rate` aggregation: `AVG` |
+| `UserDefined`   | Metric is defined by user with job parameter `mantis.jobmaster.autoscale.metric` in this format `{group}::{name}::{aggregation}`. |
+
+Each strategy has the following parameters:
+
+| Name            | Description  |
+| --------------- | ------------ |
+| `Scale down below percentage` | When the aggregated value for all workers falls below this value, the stage will scale down. It will scale down by the decrement value specified in the policy. For data drop, this is calculated as the number of data items dropped divided by the total number of data items, dropped+processed. For CPU, Memory, etc., it is calculated as a percentage of allocated resource when you defined the worker. |
+| `Scale up below percentage`   | When the aggregated value for all workers rises above this value, the stage will scale up. |
+| `Rolling count`               | This value helps to keep jitter out of the autoscaling process. Instead of scaling immediately the first time values fall outside of the scale-down and scale-up percentage thresholds you define, Mantis will wait until the thresholds are exceeded a certain number of times within a certain window. For example, a rolling count of “6 of 10” means that only if in ten consecutive observations six or more of the observations fall below the scale-down threshold will the stage be scaled down. |
+
+It is possible to employ multiple rule based strategies for a stage. In this case, as soon as 1 strategy triggers a
+scaling action, the cooldown will prevent subsequent strategies from scaling for that duration.
+
+!!! note
+    Ideally, there should be zero data drop, so there isn’t an elegant way to express “scale down
+    below percentage” for data drop. Specifying “0%” as the “scale down below percentage”
+    effectively means the data drop percentage never trigger a scale down. For this reason, it is
+    best to use the data drop strategy in conjunction with another strategy that provides the
+    scale-down trigger.
+
+## PID Control Based Strategy
+
+PID control system uses a continuous feedback loop to maintain a signal at a target level. Mantis offers variations of
+this strategy that operations on different signals. Additionally, they try to learn the appropriate target over time
```

Comment:
operations -> operates. We should also call out the limitation of fixed step size in Rule based autoscaling, which is not the case with PID control as the step size is proportional to the error(diff between set point and observed values)